Content.Shared/Labels/Components/HandLabelerComponent.cs

```diff
@@ -26,6 +27,15 @@ public sealed partial class HandLabelerComponent : Component
 
     [DataField]
     public EntityWhitelist Whitelist = new();
+
+    /// <summary>
+    /// Goobstation - sound played when applying a label
+    /// </summary>
+    [DataField]
+    public SoundPathSpecifier PrintSound = new SoundPathSpecifier("/Audio/_Goobstation/Items/hand_labeler_print.ogg")
+    {
+        Params = AudioParams.Default.WithVolume(-2f)
```

Comment:
```suggestion
        Params = AudioParams.Default.WithVolume(10f)
```
It sounded too quiet, I think?
on my end no matter what I put as long as it wasn't on minus was fine